src/mechanics/elements/ContinuumElementIGA.cpp

```diff
@@ -55,7 +56,8 @@ const Eigen::VectorXd NuTo::ContinuumElementIGA<TDim>::GetIntegrationPointVolume
     Eigen::VectorXd volume(this->GetNumIntegrationPoints());
     for (int theIP = 0; theIP < this->GetNumIntegrationPoints(); theIP++)
     {
-        Eigen::MatrixXd derivativeShapeFunctionsNatural = this->mInterpolationType->Get(Node::eDof::COORDINATES).CalculateDerivativeShapeFunctionsNatural(theIP, mKnotIDs);
+        const auto ipCoords = this->GetIntegrationType().GetLocalIntegrationPointCoordinates(theIP);
```

Comment:
http://en.cppreference.com/w/cpp/language/this

`mInterpolationType` is not a member of `ContinuumElementIGA` but from its base class `ContinuumElement` and has to be referenced with `this`.